src/components/views/dialogs/DeactivateAccountDialog.tsx

```diff
@@ -104,7 +104,7 @@ export default class DeactivateAccountDialog extends React.Component<IProps, ISt
         this.setState({ bodyText, continueText, continueKind });
     };
 
-    private onUIAuthFinished = (success: boolean, result: Error) => {
+    private onUIAuthFinished = (success, result) => {
```

Comment:
Why did we drop the types here?

Reply:
Because I was switching to `private onUIAuthFinished: InteractiveAuthCallback = (success, result) => {` but forgot to actually finish that part.